### DISCUSSION TITLE
suggestion: maglock additions

### DISCUSSION THREAD

**Discussion by luckyrada** [tsunamayo/Starship-EVO#4213] [Suggestions]
this is two suggestions in one. first, have maglocks able to maglock players. i could see this as a really fun way of jailing someone and have a really cool jail room with a floating person in it, like attack of the clones.
second, have maglocks able to be used on a gun, so you could pick up very very small ships and cargo containers, and, you could pick up players, so you could move them into one of these jail cells.

**Comment by tsunamayo** [comment]
@luckyrada I think they should be able to be used on a gun. Thanks
